addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/handler/EvohomeHandler.java

```diff
@@ -49,6 +55,26 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
+        if(getBridge() != null && getBridge().getHandler() != null){
+            EvohomeApiClient apiClient = ((EvohomeGatewayHandler) getBridge().getHandler()).getApiClient();
```

Comment:
I'll check - although my preference would be that it can't be null to save us scattering null checks everywhere.